src/convert/gnosis_safe_deposit.rs

```diff
@@ -24,7 +24,7 @@ impl From<&DbGnosisSafeDeposit> for ProtoGnosisSafeDeposit {
         dst.set_token_addr(src.token_addr().to_string());
         dst.set_amount(src.amount());
         dst.set_expected_mc_mint_tx_nonce_hex(src.expected_mc_mint_tx_nonce_hex().to_string());
-        dst.set_execution_date(src.execution_date().timestamp_nanos() as u64);
+        dst.set_execution_date(src.execution_date().timestamp_nanos_opt().unwrap() as u64);
```

Comment:
TL;DR nothing to see here, keep moving

:shrug: I generally lean toward `expect()` in code. But I can't tell if this is just dogmatism creeping in or if there is good justification for it. At one point we wanted to prevent `unwraps()` with some kind of static analyzer, but there was no good way to only run it on the code and not the tests. Using expects in tests feels noisy and overly verbose.

The reasoning is it avoids a commented unwrap
```rust
// This should never happen because blah
let bar = foo.unwrap();
```
and encourages 
```rust
let bar = foo.expect("Foo should have been stored correctly in the DB");
```

Reply:
heck yeah! I'll update those unwraps before merging.